cmake/circle_continuous.cmake

```diff
@@ -20,15 +21,22 @@ ctest_start("Continuous")
 ctest_configure(OPTIONS "${cfg_options}")
 ctest_build()
 
+# Only run the packaging tests on master branch, or a release branch
+if(branch STREQUAL "master" OR branch MATCHES "^v[0-9]+\\.[0-9]+\\.[0-9]+")
+  set(_label "girder_python|girder_package")
+else()
+  set(_label "girder_python")
+endif()
+
 if(test_group STREQUAL python)
   ctest_test(
     PARALLEL_LEVEL 4 RETURN_VALUE res
-    INCLUDE_LABEL girder_python
+    INCLUDE_LABEL "${_label}"
   )
 elseif(test_group STREQUAL browser)
   ctest_test(
     PARALLEL_LEVEL 4 RETURN_VALUE res
-    EXCLUDE_LABEL girder_python
+    EXCLUDE_LABEL "girder_python|girder_package"
```

Comment:
Sadly no, it would cause the packaging tests to be run in the browser group on all branches that didn't match.